config/default/field.field.node.page.field_after_content_override.yml

```diff
@@ -0,0 +1,29 @@
+uuid: 12008330-781b-43ed-af2e-82eb97ceecc8
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_after_content_override
+    - fragments.fragment_type.region_item_after_content
+    - node.type.page
+id: node.page.field_after_content_override
+field_name: field_after_content_override
+entity_type: node
+bundle: page
+label: 'After content override'
+description: 'Override the sitewide sticky <i>After content</i> region for this page. <a href="/admin/config/sitenow/region-settings">Manage or create <i>After content</i> items.</a>'
```

Comment:
Do we want to put `target="_blank" rel="noopener"` here?

Reply:
Good idea, I was concerned about the items not populating, but then i remembered that it is ajax fetching the items, so it's all good!

Reply:
Just added this